Rakefile

```diff
@@ -1,8 +1,9 @@
-require 'bundler/setup'
```

Comment:
With bundler/setup included in the rakefile, this *forces* bundler to be used by all developers. But it needn't be.

Alternatively, if bundler is going to be a required development dependency, it needs to be added to the gemspec.